scripts/qesap/test/unit/test_qesap_ansible.py

```diff
@@ -370,4 +370,4 @@ def test_ansible_env_config():
 fi
 export ANSIBLE_PIPELINING=True
     """
-    pass
+    return
```

Comment:
I know about this warning, but it was not so bad remembering me that one more test has to be implemented asap :-)

Reply:
> I know about this warning, but it was not so bad remembering me that one more test has to be implemented asap :-)

I believe it's better to have a TODO comment in this case rather than `pass` which is preferred in other contexts such as try/except.